gulpfile.js

```diff
@@ -17,6 +17,7 @@ var gulp = require('gulp'),
     minimatch = require('minimatch'),
     copy = require('gulp-copy'),
     concat = require('gulp-concat'),
+    promise = require('promise'),

```

Comment:
Promises are natively supported by Node.js 4 and 5 so you don't need this dependency.